src/bo4e/bo/preisblatthardware.py

```diff
@@ -0,0 +1,71 @@
+"""
+Contains PreisblattHardware class and corresponding marshmallow schema for de-/serialization
+"""
+from typing import List, Optional
+
+import attr
+from marshmallow import fields
+from marshmallow_enum import EnumField  # type:ignore[import]
+
+from bo4e.bo.preisblatt import Preisblatt, PreisblattSchema
+from bo4e.com.geraeteeigenschaften import Geraeteeigenschaften, GeraeteeigenschaftenSchema
+from bo4e.enum.bilanzierungsmethode import Bilanzierungsmethode
+from bo4e.enum.botyp import BoTyp
+from bo4e.enum.dienstleistungstyp import Dienstleistungstyp
+from bo4e.enum.netzebene import Netzebene
+
+
+# pylint: disable=too-few-public-methods
+@attr.s(auto_attribs=True, kw_only=True)
+class PreisblattHardware(Preisblatt):
+    """
+    Variante des Preisblattmodells zur Abbildung der Preise für zusätzliche Hardware
+    """
+
+    bo_typ: BoTyp = attr.ib(default=BoTyp.PREISBLATTHARDWARE)
+    # required attributes (additional to those of Preisblatt)
+    #: Die Preise gelten für Marktlokationen der angebebenen Bilanzierungsmethode
+    bilanzierungsmethode: Bilanzierungsmethode = attr.ib(validator=attr.validators.instance_of(Bilanzierungsmethode))
+    #: Die Preise gelten für Messlokationen in der angebebenen Netzebene
+    messebene: Netzebene = attr.ib(validator=attr.validators.instance_of(Netzebene))
+
+    #: Der Preis betriftt das hier angegebene Gerät, z.B. ein Tarifschaltgerät
+    basisgeraet: Geraeteeigenschaften = attr.ib(validator=attr.validators.instance_of(Geraeteeigenschaften))
```

Comment:
nur diese zeile (und entsprechungen in Schema+Test) unterscheiden diesen PR von #334